test-files/ui/features/command_detail_modal.feature

```diff
@@ -0,0 +1,31 @@
+Feature: Command detail modal window
+
+  Scenario: I can conflate the AllDataTypes data
+    Given I am on Hootenanny
+    And I resize the window
+    And I click Get Started
+    And I press "Add Reference Dataset"
+    And I click the "DcOsmRoadsCucumber" Dataset
+    And I press "Add Layer"
+    Then I wait 30 "seconds" to see "span.strong" element with text "DcOsmRoadsCucumber"
+    And I press "Add Secondary Dataset"
+    And I click the "DcTigerRoadsCucumber" Dataset
+    And I press "Add Layer"
+    Then I wait 30 "seconds" to see "span.strong" element with text "DcTigerRoadsCucumber"
+    Then I should see "Conflate"
+    And I press "Conflate"
+    And I fill "saveAs" input with "MergedDcRoads"
+    And I scroll element into view and press "conflate2"
+    Then I wait 30 "seconds" to see "Conflating …"
+    Then I wait 20 seconds
+    Then I see the UI alert
+    And I click on ".alertDiv"
+    And I click on ".detailLink"
+    Then I should see a paragraph with text "commandDetail"
```

Comment:
'commandDetail' comes from the REST call like 'status: complete' and is why I went ahead and made the modal window (as a way to visualize all of the tasks being done for a job, it was way too much data for a UI alert window), so that's why I had the test check for that. The modal window could still work properly if it got a 'status: failed' or similar.